sdk-extensions/aws/src/main/java/io/opentelemetry/sdk/extension/aws/resource/LambdaResource.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.aws.resource;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
```

Comment:
Can you rebase? I've removed the guava dependency from this module. You can use `getOrDefault` and just `isEmpty` instead I believe.

Reply:
Done.